pkg/kubelet/kubelet.go

```diff
@@ -2149,7 +2161,6 @@ func (kl *Kubelet) syncLoopIteration(configCh <-chan kubetypes.PodUpdate, handle
 			glog.Errorf("Update channel is closed. Exiting the sync loop.")
```

Comment:
Please modify `cleanupOrphanedPodDirs()`/`podVolumesExist()` to check disk for volumes as well.